content/2020-06-19-how-to-learn-typescript-quickly/index.md

```diff
@@ -0,0 +1,259 @@
+
+### Practice
+
+Learning means nothing without practice. With your new TypeScript knowledge, you can build projects or contribute to open source. This is where the application phase of the learning cycle appears. It doesn't matter what you build, but that the act of building happens.
+
+When I was learning, I followed a tutorial to [build a tiny compiler](https://github.com/jamiebuilds/the-super-tiny-compiler). It was written in JavaScript,  but I did it in TypeScript. This forced me to figure out things on my own.
+
+Other things I did include contributing to open source. Remember our key results? Here's two of them that apply here:
+
+- Contribute to TypeScript, [the language](https://github.com/microsoft/TypeScript/issues?q=is%3Aopen+is%3Aissue+label%3A"good+first+issue")
+- Contribute to [typescript-cheatsheets/react-typescript-cheatsheet](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet)
+
+This is where that happens if it hasn't already! The issues that you help with may be minor, but that's not the point.
+
+**Practice is where you learn the most**.
+
+### Build Confidence
+
+You want to build confidence in whatever you learning. This will make you feel comfortable using your new knowledge.
+
+In the case of TypeScript, you want to build **TypeScript Confidence**. There are various ways to do this including:
+
+- teaching others
+- sharing (as we said before)
+- reviewing what you've learned
+- helping others
+
+### Don't Skip Vocab
+
+Vocabulary is often overlooked. Don't skip out on it! It helps build confidence and it will solidify what you know. It's also a great tool to fight imposter syndrome. Knowing the vocabulary will allow you to build **expertise**.
+
+To learn and remember vocabulary, use a *spaced-repetition system* such as [Anki](https://apps.ankiweb.net/). I used it when I was studying TypeScript. You can use it to remember things such as:
+
+- terminology
+- concepts
+- patterns
+- syntax
+
+### Go for Depth
+
+The last thing in this phase is going for depth. Deep learning. Deep understanding.
+
+There are two techniques you can use to deepen what you learn. The first is called the [Feynman Technique](https://fs.blog/2012/04/feynman-technique/). Take a concept you are learning, such as [type inference in TypeScript](https://www.typescriptlang.org/docs/handbook/type-inference.html) and explain it as you would to a seven-year-old.
+
+The idea is that if you can't do that, you don't know the concept well enough.
+
+Note: the age of the person towhom you're explaining your learning concept can vary. The goal is for you to be able to explain something you understand to someone with little knowledge or context.
+
+The second technique I want to point out is called [Ultralearning](https://www.scotthyoung.com/blog/ultralearning/). In his book, Scott Young explains Ultralearning as,
+
+> a strategy for aggressive, self-directed learning.
+>
+> —Scott Young
```

Comment:
Should I remove this?

Wondering if Fast Framework is really my own version of ultralearning.